docs/TuxOwnerClass.md

```diff
@@ -0,0 +1,269 @@
+# Tux/React.OwnerClass
+
+## <a id="Premise"></a>Premise [#](#Premise)
+>The Owner Class is an opinionated [React](http://facebook.github.io/react/) class designed to manage state in an application.  It is also an excellent class to use as an endpoint for a client side route in the app.  The OwnerClass possesses methods that allow it to more easily connect with a store or stores and expose its properties to child components without having to cascade those properties down as per the usual React workflow.
```

Comment:
Change the two spaces after period to one. Make sure to apply this comment throughout the document.